x-pack/plugins/session_view/public/components/detail_panel_process_tab/helpers.ts

```diff
@@ -132,7 +136,8 @@ export const getDetailPanelProcess = (process: Process | null): DetailPanelProce
   }
 
   processData.executable = executables.map((exe, i) => {
-    const action = i === 0 ? 'fork' : 'exec';
+    const execEventAction = index === AUDITBEAT_INDEX ? 'executed' : 'exec';
```

Comment:
what about using constants that you defined here as well?

Reply:
I was recieving import issues so I kept the literal strings for now.